app/api/puzzle/similarBatch/similarBatchFor.ts

```diff
@@ -41,14 +41,6 @@ export const similarBatchForCompromised = async (
   persist: boolean = true,
   compromise: number = INITIAL_COMPROMISE
 ): Promise<Puzzle[]> => {
-  const candidates = await preprocessing(
-    username,
-    lastBatch,
-    clampedRating,
-    solvedArray,
-    minBatchFactor,
-    compromise
-  );
```

Comment:
this is expensive, especially for me locally with 100K puzzles, so moved It to only when the cache is fully solved - speeds up Leitner massively for me (locally)